client/checkout/api/index.js

```diff
@@ -36,6 +37,7 @@ export default class WCPayAPI {
 		if ( betas ) {
 			options.betas = betas;
 		}
+		options.apiVersion = this.apiVersion + ';link_beta=v1';
```

Comment:
Changes in this file should be refactored in a way that the `link_payments` capability drives enabling betas and adding beta headers. Otherwise, we could potentially break something.

I'd be cool with injecting capabilities similar to  `betas = []` in this method.

Reply:
indeed, this will be handled in  #3843 . Should I implement it on this branch ?

Reply:
@dpaun1985: yes, that would be great.